frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/FieldListDrawer.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import { DragDropContext } from 'react-beautiful-dnd'
```

Comment:
Thanks for helping to add more storybook tests!

Not an issue, just taking the opportunity to think about what kind of components we're trying to do UI testing for.

Generally there are three kinds of UI that we want to test:
1. Reusable components. Components such as `<FormLabel />` which is expected to be re-used on multiple places
2. Page-level views. Generally to snapshot tests on mobile vs desktop UI
3. Features. Secret key modals, csv upload flow for conditional routing.

Reply:
I added this test for Alicia to verify that the `Use for approvals` badge is displayed for MRF forms only and not for other modes.